ga4gh/server/datamodel/bio_metadata.py

```diff
@@ -46,9 +45,10 @@ def toProtocolElement(self):
             individual_id=self.getIndividualId(),
             name=self.getName(),
             disease=disease)
-        for key in self.getInfo():
-            for value in self.getInfo()[key]['values']:
-                biosample.info[key].values.add().string_value = value
+        attributes = self.getAttributes()
+        for key in attributes:
+            biosample.attributes.attr[key] \
+                .values.extend(protocol.encodeValue(attributes[key]))
```

Comment:
Wonder if this pattern should be abstracted onto the `datamodel` class since it's used in all the toProtocolElement calls?
Just something like:
```
def serializeAttributes(self, msg):
   attributes = self.getAttributes()
   for key in attributes:
       msg.attributes.attr[key].values.extend(protocol.encodeValue(attributes[key]))
```